src/includes/sourcemaps/upload/primer/javascript.remix.mdx

```diff
@@ -0,0 +1,11 @@
+On release, call `sentry-upload-sourcemaps` to upload source maps and create a release. 
```

Comment:
Let's refer to this as `yarn sentry-upload-sourcemaps` (or npm version of this) as using it as `sentry-upload-sourcemaps` won't work.